app/login.tsx

```diff
@@ -0,0 +1,263 @@
+import React, { useState } from 'react';
+import {
+  View,
+  Text,
+  StyleSheet,
+  SafeAreaView,
+  TextInput,
+  TouchableOpacity,
+  KeyboardAvoidingView,
+  Platform,
+  TouchableWithoutFeedback,
+  Keyboard
+} from 'react-native';
+import { Ionicons } from '@expo/vector-icons';
+import { router } from 'expo-router';
+
+export default function LoginScreen() {
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [showPassword, setShowPassword] = useState(false);
+  const [isEmailFocused, setIsEmailFocused] = useState(false);
+  const [isPasswordFocused, setIsPasswordFocused] = useState(false);
+
+  const handleLogin = () => {
+    // Implement login logic here
+    console.log('Login with:', email, password);
+    // On successful login:
+    router.replace('/(tabs)');
+  };
+
+  return (
+    <TouchableWithoutFeedback onPress={Keyboard.dismiss}>
+      <SafeAreaView style={styles.container}>
+        <KeyboardAvoidingView
+          behavior={Platform.OS === 'ios' ? 'padding' : 'height'}
+          style={styles.keyboardAvoidView}
+        >
+          <View style={styles.headerContainer}>
+            <Text style={styles.welcomeText}>Welcome to</Text>
+            <Text style={styles.appNameText}>iWetMyPlants</Text>
+            <Text style={styles.taglineText}>Your personal plant care companion</Text>
+          </View>
+
+          <View style={styles.formContainer}>
+            <View style={[
+              styles.inputContainer,
+              isEmailFocused && styles.inputContainerFocused
+            ]}>
+              <Ionicons name="mail-outline" size={22} color="#76A97F" style={styles.inputIcon} />
+              <TextInput
+                style={styles.input}
+                placeholder="Email"
+                value={email}
+                onChangeText={setEmail}
+                autoCapitalize="none"
+                keyboardType="email-address"
+                onFocus={() => setIsEmailFocused(true)}
+                onBlur={() => setIsEmailFocused(false)}
+              />
+            </View>
+
+            <View style={[
+              styles.inputContainer,
+              isPasswordFocused && styles.inputContainerFocused
+            ]}>
+              <Ionicons name="lock-closed-outline" size={22} color="#76A97F" style={styles.inputIcon} />
+              <TextInput
+                style={styles.input}
+                placeholder="Password"
+                value={password}
+                onChangeText={setPassword}
+                secureTextEntry={!showPassword}
+                onFocus={() => setIsPasswordFocused(true)}
+                onBlur={() => setIsPasswordFocused(false)}
+              />
+              <TouchableOpacity
```

Comment:
Touchable opacity works but I recommend using pressables instead because they offer more functionality and props. Heres more info on the difference [pressables ](https://www.dhiwise.com/post/ultimate-showdown-react-native-touchableopacity-vs-pressable) if you're interested